README.md

```diff
@@ -16,24 +16,142 @@ such as when a branch is pushed to.
 This is useful for creating a continuous deployment infrastructure, which runs tests
 or deploys software when it is updated.
 
+## The problem with web hooks
+
 The problem with webhooks is that it requires that the receiver of the hook listens on
```

Comment:
This needs plural agreement: "The problem with webhooks are that they require that..."

Optionally, you could simplify it by "Webhooks require that ...": you've already said in the header that they are a problem.